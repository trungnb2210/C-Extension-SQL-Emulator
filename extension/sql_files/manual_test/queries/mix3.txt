SELECT item, MIN(price)
FROM things JOIN prices ON object == item
GROUP BY item
HAVING MIN(price) < 150
ORDER BY item, MIN(price)
