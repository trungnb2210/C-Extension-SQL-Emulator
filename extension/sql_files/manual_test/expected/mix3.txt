QUERY RESULT:
  item   MIN(price) 
---------------------
| Box  | 100        | 
| Tire | 120        | 
---------------------
